eng/sdk/sync_sdk.py

```diff
@@ -137,6 +143,9 @@ def update_sdks():
     cmd = ['git', 'add', '.']
     subprocess.check_call(cmd, cwd=sdk_root)
 
+    if failed_artifact:
+        logging.error(f'Failed modules {failed_artifact}')
```

Comment:
Though, generally we actually want a failure. This can be done here by re-throwing an error.